src/rules/no-namespace.js

```diff
@@ -17,12 +17,30 @@ module.exports = {
       url: docsUrl('no-namespace'),
     },
     fixable: 'code',
-    schema: [],
+    schema: [{
+      type: 'object',
+      properties: {
+        ignoreFromPattern: {
+          type: 'string',
+        },
+      },
+    }],
   },
 
   create: function (context) {
+    const firstOption = context.options[0] || {};
+    const ignoreFromPatternExpression =
+      firstOption.ignoreFromPattern &&
+      new RegExp(firstOption.ignoreFromPattern);
```

Comment:
eslint rules should never accept regex patterns; this is a common source of ReDOS CVEs. Globs are the only available choice here.

Reply:
Hi! I rewrote it to use minimatch instead of regex.